Bio/GenBank/__init__.py

```diff
@@ -523,7 +523,8 @@ def _unhandled(self, data):
     def __getattr__(self, attr):
         return self._unhandled
 
-    def _split_keywords(self, keyword_string):
+    @staticmethod
+    def _split_keywords(keyword_string):
```

Comment:
Good idea. As I recall when this was written we didn't have ``@staticmethod`` in Python, but we can use it now.